src/snapshot_middleware/lua.rs

```diff
@@ -20,36 +36,54 @@ pub fn snapshot_lua(
 ) -> anyhow::Result<Option<InstanceSnapshot>> {
     let file_name = path.file_name().unwrap().to_string_lossy();
 
-    let (class_name, instance_name) = if let Some(name) = match_trailing(&file_name, ".server.lua")
+    let is_run_context_enabled = true; //project.script_type == ScriptType::RunContext;
```

Comment:
```suggestion
    // project.script_type == ScriptType::RunContext;
    let is_run_context_enabled = true;
```
We generally prefer to put comments above code